test/cvss.spec.js

```diff
@@ -49,3 +49,37 @@ describe("Rating Tests", () => {
     expect(vector.getRating()).toBe("Critical");
   });
 });
+
+describe("Rating Tests", () => {
```

Comment:
Could you please change the name of the test to something like `Vector Object Tests`?

Reply:
Oh damn I had changed that... Good catch. It must've changed while I was undoing something hahahah.
I'm so sorry. I'll do it right away!

Thanks

Reply:
No worries!